src/components/UserDisplay.vue

```diff
@@ -5,8 +5,7 @@
           <v-list-item-avatar>
             <v-img :src="getImg !== null ? 'https://imgplaceholder.com/420x320/3a47c6/f7f7f7/glyphicon-user' : getimg " alt="avatar"></v-img>
           </v-list-item-avatar>
-
-          <v-list-item-content>
+            <v-list-item-content>
```

Comment:
Your change was adding a space which shouldn't be there
```suggestion
           <v-list-item-content>
```